server/task_executor_schema_update_ghost_sync.go

```diff
@@ -120,28 +152,33 @@ func newMigrationContext(config ghostConfig) (*base.MigrationContext, error) {
 	return migrationContext, nil
 }
 
-// RunOnce will run SchemaUpdateGhostSync task once.
-func (exec *SchemaUpdateGhostSyncTaskExecutor) RunOnce(ctx context.Context, server *Server, task *api.Task) (terminated bool, result *api.TaskRunResultPayload, err error) {
-	payload := &api.TaskDatabaseSchemaUpdateGhostSyncPayload{}
-	if err := json.Unmarshal([]byte(task.Payload), payload); err != nil {
-		return true, nil, fmt.Errorf("invalid database schema update gh-ost sync payload: %w", err)
-	}
-	return runGhostMigration(ctx, exec.l, server, task, db.Migrate, payload.Statement, payload.SchemaVersion, payload.VCSPushEvent)
-}
-
 func runGhostMigration(ctx context.Context, l *zap.Logger, server *Server, task *api.Task, migrationType db.MigrationType, statement, schemaVersion string, vcsPushEvent *vcsPlugin.PushEvent) (terminated bool, result *api.TaskRunResultPayload, err error) {
 	mi, err := preMigration(ctx, l, server, task, migrationType, statement, schemaVersion, vcsPushEvent)
 	if err != nil {
 		return true, nil, err
 	}
-	migrationID, schema, err := executeSync(ctx, l, task, mi, statement)
-	if err != nil {
-		return true, nil, err
-	}
-	return postMigration(ctx, l, server, task, vcsPushEvent, mi, migrationID, schema)
+
+	waitSync := &sync.WaitGroup{}
```

Comment:
IMO `waitCutover` is better, because this `WaitGroup` is waiting for gh-ost cutover to start.

Reply:
We're waiting for the ghost table to sync with the original table; not quite waiting for cutover, IMO.